package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "inno-test",
```

Comment:
should be dsrm-forntend

Reply:
Fixed in d3482ab3cd3bc0279ad198369012a9fd8b289138